topics/ecology/tutorials/phylodiversity_workflow/tutorial.md

```diff
@@ -0,0 +1,200 @@
+# Before starting
+
+This part will present the type of data you need to run the ecoregionalization workflow. This data will be downloaded in the next part of the tutorial.
+
+## phylogenic tree file
+The first file needed for this workflow is the phylogenetic tree of your interested species.
+In this example it'a a simplified phylogeny of the actinopterigy
+This file must be at newick format.
+
+## occupancy file
+The second file is an occupancy file, each line is a species, the decimal separator must be "." and the column must be separated with "\t"(={Tabulation}). You need to have a column "grids" containing the cell of the grid you've seen your species and the name of the column with the species names must be "newscientificname".
+
+| grids   | newscientificname   | ...     | ...   | ...   |
+| ------- | ------------------- | ------- | ----- | ----- |
+| ------- | ------------------- | ------- | ----- | ...   |
+| ------- | ------------------- | ------- | ----- | ----- |
+| ...     | ...                 | ...     | ...   | ...   |
+| ------- | ------------------- | ------- | ----- | ----- |
+
+## Shapefile
+The last file is a spatial file in shapefile format. In Galaxy this type of file must be uploaded as a composite file of type shp.
+This kind of file must have at least 3 file with the same name and 3 different extension : .shp, .shx end .dbf. you can have more file optionally like the .prj file.
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/api/records/15487657/files/shapefile.dbf/content
+>    https://zenodo.org/api/records/15487657/files/shapefile.prj/content
+>    https://zenodo.org/api/records/15487657/files/phylogeny_test/content
+>    https://zenodo.org/api/records/15487657/files/shapefile.shx/content
+>    https://zenodo.org/api/records/15487657/files/shapefile.shp/content
+>    https://zenodo.org/api/records/15487657/files/composite_dataset_shp.html/content
+>    https://zenodo.org/api/records/15487657/files/grid_test.csv/content
+>    ```
+>
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets
+> 4. Check that the datatype of the phylogenic phyle is newick, the occupancy file is tabular and the shape file is a composite dataset of type shapefile
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add to each database a tag corresponding to ...
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+# Data formatting
+
+The first step is to be sure your data are well formated. If all your file are in good format and do have the needed column as specified before, you can move forward.
```

Comment:
maybe add the first few lines of the file here so that people can compare their file to what it should look like?

Reply:
I tried to add a screenshot of an example data file